packages/faceted-search/src/components/BasicSearch/BasicSearch.scss

```diff
@@ -2,7 +2,7 @@
 	align-items: center;
 	display: flex;
 	height: inherit;
-	overflow: auto;
+	overflow: visible;
```

Comment:
This introduces a new behavior: no scroll anymore. If you have tons of badges, it will just take more place, instead of scrolling.
Is it something that is validated with UX ?

(Not related, but a question: why do we have a keight: inherit just above ?)

Reply:
I append some badges, and then reduce the width of the browser,  it wraps the badges to next line, is it acceptable?

Reply:
☝️ `Is it something that is validated with UX ?`

Reply:
I asked Maxime Nicoul, and waiting for his reponse.

Reply:
@jsomsanith  I confirmed with  Maxime Nicoul, when set it to "visible", with many badges, it would wrap to multi-line to display all filters,  this behavior is acceptable.